lib/grape/dsl/inside_route.rb

```diff
@@ -180,8 +180,9 @@ def error!(message, status = nil, additional_headers = nil)
       # A Rack::Response object containing the specified message, status, and headers.
       #
       def rack_response(message, status = 200, headers = { Rack::CONTENT_TYPE => content_type })
-        message = ERB::Util.html_escape(message) if headers[Rack::CONTENT_TYPE] == 'text/html'
-        Rack::Response.new([message], Rack::Utils.status_code(status), headers)
+        Grape.deprecator.warn('Use error! instead of rack_response')
```

Comment:
I think this reads cleaner?

The rack_response method has been deprecated, use error! instead.